numpy/lib/_npyio_impl.py

```diff
@@ -523,10 +523,12 @@ def save(file, arr, allow_pickle=True, fix_imports=True):
         compatible between different versions of Python).
         Default: True
     fix_imports : bool, optional
-        Only useful in forcing objects in object arrays on Python 3 to be
-        pickled in a Python 2 compatible way. If `fix_imports` is True, pickle
-        will try to map the new Python 3 names to the old module names used in
-        Python 2, so that the pickle data stream is readable with Python 2.
+        The `fix_imports` flag is deprecated and has no effect.
+        .. deprecated:: 2.0
+            Historically, this flag was used to control compatibility support
+            for objects saved in Python 3 to be loadable in Python 2. This flag
+            is ignored after NumPy 1.17, and deprecated in NumPy 2.0. It will
+            be removed in a future release.
```

Comment:
Looks good as is, might suggest to shorten it a bit: This flag is ignored since NumPy 1.17 and was only needed to support loading some files in Python 2 written in Python 3.